app/models/export/submissions/row.rb

```diff
@@ -56,9 +56,11 @@ def po_number
         submission.purchase_order_number
       end
 
-      # Fields that are nil for MVP
-      def created_by; end
+      def created_by
+        submission.created_by.id
```

Comment:
So probably wasn't clear from the story, but we need to expose the user's name here, rather than their ID as CCS want to include the submitter name on the invoices they generate, and they have no way of determining that from the ID.